ambry-cloud/src/main/java/com/github/ambry/cloud/CloudMessageFormatWriteSet.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2022 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.cloud;
+
+import com.github.ambry.cloud.CloudBlobStore.CloudWriteChannel;
+import com.github.ambry.messageformat.MessageFormatWriteSet;
+import com.github.ambry.store.MessageInfo;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.atomic.AtomicLong;
+
+
+public class CloudMessageFormatWriteSet extends MessageFormatWriteSet {
```

Comment:
Changes to add this class conflicts with existing changes in PR [2076](https://github.com/linkedin/ambry/pull/2076/files#diff-434e679f770d554421fd258372c7abe92645f58b14e94865ebe65b52e8b69349). Will resolve it once the other PR is merged.